lara-typescript/src/section-authoring/components/section-item-plugin-list.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+import { ICreatePageItem, SectionColumns } from "../api/api-types";
+import { UserInterfaceContext } from "../containers/user-interface-provider";
+import { useGetAvailablePlugins, useGetPageItemEmbeddableMetaData,
+         useGetPageItemPlugins, usePageAPI } from "../hooks/use-api-provider";
+import { SectionItemPlugin } from "./section-item-plugin";
+
+import "./section-item-plugin-list.scss";
+
+interface ISectionItemPluginListProps {
+  sectionColumn: SectionColumns;
+  sectionId: string;
+  sectionItemId: string;
+}
+
+export const SectionItemPluginList: React.FC<ISectionItemPluginListProps> = ({
+  sectionColumn,
+  sectionId,
+  sectionItemId
+  }: ISectionItemPluginListProps) => {
+  const availablePlugins = useGetAvailablePlugins();
+  const pageItemPlugins = useGetPageItemPlugins(sectionItemId);
+  const pageItemEmbeddableMetaData = useGetPageItemEmbeddableMetaData(sectionItemId);
+  const { userInterface: {editingItemId}, actions: {setEditingItemId}} = React.useContext(UserInterfaceContext);
```

Comment:
If you wanted to you could extract `editingItemId` ,`wrappedItemId`, `setEditingItemId`, and `setWrappedItemId` into the same destructured assignment here...